blocks/editable/index.js

```diff
@@ -436,7 +436,7 @@ export default class Editable extends Component {
 		// Find the parent "relative" positioned container
 		const container = this.props.inlineToolbar ?
 			this.editor.getBody().closest( '.blocks-editable' ) :
-			this.editor.getBody().closest( '.editor-block-list__block' );
+			this.editor.getBody().closest( '.editor-block-item' );
```

Comment:
We should definitely try to find an alternative here, we're referencing a classname created in another module.